moveit_ros/warehouse/warehouse/src/planning_scene_storage.cpp

```diff
@@ -266,9 +266,9 @@ void moveit_warehouse::PlanningSceneStorage::getPlanningQueriesNames(std::vector
   q->append(PLANNING_SCENE_ID_NAME, scene_name);
   std::vector<MotionPlanRequestWithMetadata> planning_queries = motion_plan_request_collection_->queryList(q, true);
   query_names.clear();
-  for (std::size_t i = 0; i < planning_queries.size(); ++i)
-    if (planning_queries[i]->lookupField(MOTION_PLAN_REQUEST_ID_NAME))
-      query_names.push_back(planning_queries[i]->lookupString(MOTION_PLAN_REQUEST_ID_NAME));
+  for (MotionPlanRequestWithMetadata& planning_querie : planning_queries)
```

Comment:
```suggestion
  for (MotionPlanRequestWithMetadata& planning_query : planning_queries)
```
Another -ies mishap.

Reply:
I missed this too. Corrected.